examples/plotting/plotting_blank_map.py

```diff
@@ -0,0 +1,58 @@
+"""
+=============================
+Plot positions on a blank map
+=============================
+
+This example showcases how to plot positions on a blank map.
+It is commonly seen in papers and presentations to show HPC positions of events that occurred at
+different times and therefore no single observation is appropriate.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import frames
+
+################################################################################
+# First we will create a blank map using with an array of zeros.
+# Since there is no WCS information, we will need to construct a header to pass to Map.
+data = np.zeros((1000, 1000))
```

Comment:
This is a reasonably large array, when it really doesn't need to be. A (10, 10) array would suffice, as you just scale the pixel size.

Also as I mentioned below I think this should be an array of `NaN`:
```suggestion
data = np.zeros((1000, 1000)) * np.nan
```

Reply:
Ah, an array full of `NaN` makes me happier.  In that case, definitely go back to using `np.full()`:
```suggestion
data = np.full((1000, 1000), np.nan)
```